pkg/services/ngalert/notifier/redis_peer_test.go

```diff
@@ -45,6 +45,35 @@ func TestNewRedisPeerClusterMode(t *testing.T) {
 	require.NoError(t, ping.Err())
 }
 
+func TestNewRedisPeerSentinelMode(t *testing.T) {
```

Comment:
Maybe use https://github.com/Bose/minisentinel in here for a stronger test?

Reply:
good call! done in https://github.com/grafana/grafana/pull/106322/commits/3c2ec8aa3b9f1c1737699e4671a4cab9fdce3ecb.

the only caveat is that minisentinel doesn't seem to support TLS, so the test only checks a Sentinel setup without TLS.

Reply:
Yeah that's fine, we have another test to check TLS.